docker/centos-local-build/run.sh

```diff
@@ -0,0 +1,6 @@
+docker build -t xpansion/centos7 .
+docker run \
+--mount type=bind,src=/home/marechaljas/CLionProjects/antares-xpansion,dst=/mnt/sources \
+--mount type=bind,src=/home/marechaljas/CLionProjects/docker_caches,dst=/mnt/caches \
```

Comment:
you put your folder names here, was it intentional as an example? in that case, maybe add a mention in the readme